src/components/RecNode/RecNode.tsx

```diff
@@ -41,10 +42,24 @@ const RecNode: FC<RecNodeProps> = ({ pathwayState, isActionable }) => {
       {isActionable && (
         <form className={classes.commentsForm}>
           <label>Comments:</label>
-          <button className={indexClasses.button} onClick={(e): void => e.preventDefault()}>
+          <button
+            className={indexClasses.button}
+            onClick={(e): void => {
+              e.preventDefault();
+              setComments(
+                comments +
```

Comment:
I don't think you want to append the default text, but just use it.  If I click it multiple times it keeps adding which seems unwanted.

![Screen Shot 2020-02-05 at 3 39 52 PM](https://user-images.githubusercontent.com/41651655/73881141-d1466080-482d-11ea-8a9c-7c48fac72680.png)

Reply:
We want to the text to append to already present text, not sure how to avoid the issue of having it fill the box multiple times on multiple clicks.

Reply:
Yeah we do want it to append not totally replace. That's how the current design works. Quick & dirty option to not multiple append is `if (!comments.includes(defaultText)) setComments(comments + defaultText)`

Reply:
sounds good to me